share/lib/hoc/import3d/read_swc.hoc

```diff
@@ -184,34 +184,96 @@ printf("  %d: %s\n", iline.x[i], lines.o(iline.x[i]).s)
 	}
 }
 
-proc sectionify() {local i, si  localobj tobj
+func check_soma3geom() {local i, nsoma_pts  localobj nchild
```

Comment:
nitpick : can we move docstring about this function here instead of call site?

Reply:
Done